shared-plugins/new-device-notification/new-device-notification.php

```diff
@@ -168,30 +168,32 @@ public function ip_to_city( $ip ) {
 		if ( ! function_exists( 'ip2location' ) ) {
 			$location = new stdClass();
```

Comment:
We could set the country code, but we currently don't have any of the other details available on VIP Go.